IOS/SwiftSeedProject/SwiftSeedProject/Interfaces/Components/NewsCollection/Cells/NewsArticleViewModel.swift

```diff
@@ -0,0 +1,29 @@
+//
+//  NewsArticleViewModel.swift
+//  SwiftSeedProject
+//
+//  Created by Lucas Pelizza on 4/18/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+import Bond
+
+final class NewsArticleViewModel {
```

Comment:
I would name it ArticleViewModel as it is a ViewModel of Article. What do you think? Just a suggestion